index.css

```diff
@@ -35,3 +46,17 @@ a:active, a:visited, a:link {
   text-decoration: none;
   color: #009688;
 }
+
+.footer{
+    width: 100%;
+    height: 50px;
+    position: fixed;
+    bottom: 0px;
+    text-align: center;
```

Comment:
it is not vertical center aligned.
Try use flexbox to do both horizontal and vertical align https://philipwalton.github.io/solved-by-flexbox/demos/vertical-centering/

Reply:
Which  is not vertical aligned?